tests/test_meshing_utilities.py

```diff
@@ -11,7 +11,6 @@ def pytest_approx(expected):
 
 @pytest.mark.skip("Activate with: https://github.com/ansys/pyfluent/pull/3205")
```

Comment:
@hpohekar, this test is getting skipped now, right? We need to activate it? And should it run for all PRs or only for nightly?

Reply:
@prmukherj I think this is skipped after pulling latest changes from main branch, I will enable it again. We can run it for all PRs as it takes less than a minute.